contrib/kind.sh

```diff
@@ -65,41 +80,108 @@ print_params()
      echo "KIND_HA = $KIND_HA"
      echo "KIND_CONFIG_FILE = $KIND_CONFIG "
      echo "KIND_REMOVE_TAINT = $KIND_REMOVE_TAINT"
+     echo "KIND_IPV4_SUPPORT = $KIND_IPV4_SUPPORT"
+     echo "KIND_IPV6_SUPPORT = $KIND_IPV6_SUPPORT"
      echo ""
 }
 
 parse_args $*
 
-MASTER_COUNT=1
+# Set default values
+KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-ovn}
 K8S_VERSION=${K8S_VERSION:-v1.17.2}
 KIND_INSTALL_INGRESS=${KIND_INSTALL_INGRESS:-false}
 KIND_HA=${KIND_HA:-false}
-if [ "$KIND_HA" == true ]; then
-  DEFAULT_KIND_CONFIG=./kind-ha.yaml
-  MASTER_COUNT=`grep -c "^\s-\srole\s*:\s*control-plane" kind-ha.yaml`
-else
-  DEFAULT_KIND_CONFIG=./kind.yaml
-  MASTER_COUNT=`grep -c "^\s-\srole\s*:\s*control-plane" kind.yaml`
-fi
-KIND_CONFIG=${KIND_CONFIG:-$DEFAULT_KIND_CONFIG}
+KIND_CONFIG=${KIND_CONFIG:-./kind.yaml.j2}
 KIND_REMOVE_TAINT=${KIND_REMOVE_TAINT:-true}
+KIND_IPV4_SUPPORT=${KIND_IPV4_SUPPORT:-true}
+KIND_IPV6_SUPPORT=${KIND_IPV6_SUPPORT:-false}
+
+# Input not currently validated. Modify outside script at your own risk.
+NET_CIDR_IPV4=${NET_CIDR_IPV4:-10.244.0.0/16}
+SVC_CIDR_IPV4=${SVC_CIDR_IPV4:-10.96.0.0/12}
+NET_CIDR_IPV6=${NET_CIDR_IPV6:-fd01::/24}
+SVC_CIDR_IPV6=${SVC_CIDR_IPV6:-fd02::/24}
```

Comment:
The current service network mask is limited in Kubernetes to 20 bits, that means it should be a /112 maximum